src/utils.ts

```diff
@@ -31,9 +36,7 @@ export function mergeGlobalProperties(
     mocks: { ...configGlobal.mocks, ...mountGlobal.mocks },
     config: { ...configGlobal.config, ...mountGlobal.config },
     directives: { ...configGlobal.directives, ...mountGlobal.directives },
-    renderStubDefaultSlot: !!(mountGlobal.renderStubDefaultSlot !== undefined
-      ? mountGlobal.renderStubDefaultSlot
-      : configGlobal.renderStubDefaultSlot)
```

Comment:
This library has three point where we can write renderStubDefaultSlot option.

- mount function
- config.global object
- config object

The last one,  I wasn't able to find  in document, but it exists in GlobalConfigOptions.
So, I fixed code to use one of them (priority: mount -> config -> config.global).

Reply:
Or if it's for internal, I'll fix my code 🙇🏻

Reply:
This makes sense, nice job figuring this out.

Reply:
Thanks a lot !!
It's an honor to hear you say that. 🙇🏻